tests/functional/Migration.php

```diff
@@ -1271,7 +1271,71 @@ function () {
         // Clean DB after test
         $DB->delete('glpi_profilerights', [
             'name' => [
-                'test_updateright_1', 'test_updateright_2', 'test_updateright_3'
+                'test_replaceright_1', 'test_replaceright_2', 'test_replaceright_3'
+            ]
+        ]);
+    }
+
+    public function testGiveRight()
```

Comment:
This test should be improved to validate that the `$requiredrights` it correctly used to targets the corresponding profiles.